sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1979,6 +1979,13 @@ object SQLConf {
       .doc("When true, the ArrayExists will follow the three-valued boolean logic.")
       .booleanConf
       .createWithDefault(true)
+
+  private[spark] val ADDITIONAL_REMOTE_REPOSITORIES =
+    ConfigBuilder("spark.sql.additionalRemoteRepositories")
+      .doc("A comma-delimited string config of the optional additional remote maven mirror " +
+        "repositories, this can be used for downloading Hive jars in IsolatedClientLoader.")
+      .stringConf
+      .createWithDefault("")
```

Comment:
We're not going to set the default option, can we use `createOptional` instead?

Reply:
This is kind of an array option, it's ok to use empty string as default, like `DISABLED_V2_STREAMING_WRITERS`.

Reply:
Alright it's at least consistent with other instances. But I think we should strictly use `createOptional` if that option is optional at least to make it easier to read and consistent.